### PR TITLE
Fixing link to headless recorder

### PR DESCRIPTION
**Pull request by spekulatius**
Hey @angrykoala 

just fixing the link to the headless recorder docs.

Cheers!

### CODE REVIEW

**Comment by angrykoala** [comment]
Hi @spekulatius, thanks for your PR

As I see the change in the website comes from a rename on this tool to support playwright, Could you change the name and description of the item in the list as well to reflect these changes please?
Thanks!

**Comment by spekulatius** [comment]
Hey @angrykoala 

done, have a look and let me know if you need any further tweaks.

Cheers!

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Browser automation is the act of executing actions automatically in a web browse
   * [Browserless](https://browserless.js.org) - Complementary library for Puppeteer with sensible defaults, adblocker and pooling.
   * [Puppeteer-Firefox](https://github.com/puppeteer/puppeteer/tree/master/experimental/puppeteer-firefox) - Experimental support for Firefox using Puppeteer.
   * [Puppeteer-Extra](https://github.com/berstend/puppeteer-extra/tree/master/packages/puppeteer-extra) - Plugin support for Puppeteer and several plugins including reCAPTCHA and adblocker.
-  * [Puppeteer-Recorder](https://www.checklyhq.com/docs/headless-recorder) - Chrome extension to record interactions and generate Puppeteer scripts.
+  * [Headless Recorder](https://www.checklyhq.com/docs/headless-recorder) - Chrome extension that records your browser interactions and generates a Puppeteer or Playwright script.
```
Comment (angrykoala) [inline README.md:39]: Everything looking good here :+1:

File: README.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ Browser automation is the act of executing actions automatically in a web browse
 
 ## Tools
 
-:lock: - Private  
-:moneybag: - Paid  
-:no_entry_sign: - Not Maintained  
+:lock: - Private
```
Comment (angrykoala) [inline README.md:17]: I'm not sure of the reasoning behind this change,but the list is a bit harder to read this way, please add the trailing spaces back

**Review by angrykoala — CHANGES_REQUESTED** [review]
a minor change that needs to be reverted

File: README.md (hunk 2 above)
Comment (spekulatius) [reply README.md:17]: Ah, that's my editor - it's removing trailing spaces automatically... One second

Comment (spekulatius) [reply README.md]: Done @angrykoala

**Comment by spekulatius** [comment]
Thanks for working through this :+1:  @angrykoala

**Pull request merged**